plugins/multiserver/CMakeLists.txt

```diff
@@ -2,7 +2,8 @@ set(DESCRIPTORS
     com_osvr_Multiserver_OneEuroFilter
     com_osvr_Multiserver_OSVRHackerDevKit
     com_osvr_Multiserver_RazerHydra
-    com_osvr_Multiserver_YEI_3Space_Sensor)
+    com_osvr_Multiserver_YEI_3Space_Sensor
+	com_osvr_Multiserver_Sensics_zSight)
```

Comment:
spacing looks like it's not quite matching in the diff, but this is a silly picky thing and I can deal with it next time I touch this plugin if you like.